cl/search/fixtures/test_objects_search.json

```diff
@@ -99,7 +99,7 @@
       "case_name_short": "Howard",
       "blocked": false,
       "syllabus": "some rando syllabus",
-      "date_filed": "1795-06-09",
+      "date_filed": "1895-06-09",
```

Comment:
This was a strange fix, and I did this a few weeks ago, so I unfortunately don't remember what the problem was exactly. At bottom, though, having this fixture being filed at 1795 was not matching properly, so I changed it to 1895. I believe my guess was that this had something to do with recent changes to `reporters_db` -- something about the reporter associated with the citation associated with this opinion not being active in 1795? I don't think I fully figured it out, but this change was necessary to make the tests pass.

Reply:
Thanks for calling this out. This change broke a separate test, but knowing you changed that made fixing that test easy.